folly/docs/FBVector.md

```diff
@@ -48,7 +48,7 @@ was not a good choice: it can be mathematically proven that a growth factor of
 2 is rigorously the <i>worst</i> possible because it never allows the vector 
 to reuse any of its previously-allocated memory. Despite other compilers
 reducing the growth factor to 1.5, gcc has staunchly maintained its factor of
-2. This makes `std::vector` cache- unfriendly and memory manager unfriendly.
+2. This makes `std::vector` cache-unfriendly and memory manager unfriendly.
```

Comment:
Only this one is a real typo. All of the others appear to be artifacts of the chosen horizontal ruler, which the changes then violate.